Aquality.WinAppDriver.Template/Aquality.WinAppDriver.Template/Forms/CalculatorForm.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Forms;
+using OpenQA.Selenium;
+using System.Collections.Generic;
+
+namespace Aquality.WinAppDriver.Template.Forms
+{
+    public class CalculatorForm : Form
+    {
+        private static readonly IDictionary<Operation, string> OperationsLocators = new Dictionary<Operation, string>
+        {
+            { Operation.Add, "+" },
+            { Operation.Equals, "=" }
+        };
+
+        private ILabel ResultsLabel => ElementFactory.GetLabel(By.XPath("//*[@AutomationId='48']"), "Results bar");
+        
+        private IButton AboutTheAuthorButton => ElementFactory.GetButton(By.XPath("//*[@AutomationId='30']"), "About the Author");
```

Comment:
xpath locators are not very productive in general. For AutomationId attributes it's better to use MobileBy.AccessibilityId("30")